csl-data.json

```diff
@@ -1,447 +1,440 @@
+                    "type": "string"
+                },
+                "dimensions": {
+                    "type": "string"
+                },
+                "DOI": {
+                    "type": "string"
+                },
+                "event": {
+                    "type": "string"
+                },
+                "event-place": {
+                    "type": "string"
+                },
+                "first-reference-note-number": {
+                    "type": "string"
+                },
+                "genre": {
+                    "type": "string"
+                },
+                "ISBN": {
+                    "type": "string"
+                },
+                "ISSN": {
+                    "type": "string"
+                },
+                "jurisdiction": {
+                    "type": "string"
+                },
+                "keyword": {
+                    "type": "string"
+                },
+                "locator": {
+                    "type": "string"
+                },
+                "medium": {
+                    "type": "string"
+                },
+                "note": {
+                    "type": "string"
+                },
+                "original-publisher": {
+                    "type": "string"
+                },
+                "original-publisher-place": {
+                    "type": "string"
+                },
+                "original-title": {
+                    "type": "string"
+                },
+                "page": {
+                    "type": "string"
+                },
+                "page-first": {
+                    "type": "string"
+                },
+                "PMID": {
+                    "type": "string"
+                },
+                "PMCID": {
+                    "type": "string"
+                },
+                "publisher": {
+                    "type": "string"
+                },
+                "publisher-place": {
+                    "type": "string"
+                },
+                "references": {
+                    "type": "string"
+                },
+                "reviewed-title": {
+                    "type": "string"
+                },
+                "scale": {
+                    "type": "string"
+                },
+                "section": {
+                    "type": "string"
+                },
+                "source": {
+                    "type": "string"
+                },
+                "status": {
+                    "type": "string"
+                },
+                "title": {
+                    "type": "string"
+                },
+                "title-short": {
+                    "type": "string"
+                },
+                "URL": {
+                    "type": "string"
+                },
+                "version": {
+                    "type": "string"
+                },
+                "year-suffix": {
```

Comment:
This is a property of the item in rendered context. Any input value would be ignored.

Reply:
So can this be removed?